modules/electromagnetics/include/bcs/VectorTransientAbsorbingBC.h

```diff
@@ -30,6 +30,9 @@ class VectorTransientAbsorbingBC : public VectorIntegratedBC
   /// Intrinsic impedance of the infinite medium (default is the admittance of free space)
   const Function & _admittance;
 
+  /// Magnetic permeability of free space in SI units (H/m)
+  const Real _mu_vacuum;
```

Comment:
small nitpick: _mu0 is a bit of a more standard way of writing this no?

Reply:
You're not wrong. I wanted to be a bit verbose here for those not used to the standard naming, but I think the comment in the header makes it pretty clear by itself. Don't mind changing it.